kolibri/utils/env.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import sys
+from distutils import util
+
+logger = logging.getLogger(__name__)
+
+
+def get_cext_path(dist_path):
+    """
+    Get the directory of dist/cext.
+    """
+    # Python version of current platform
+    python_version = 'cp' + str(sys.version_info.major) + str(sys.version_info.minor)
+    dirname = os.path.join(dist_path, 'cext/' + python_version)
+
+    platform = util.get_platform()
+    # For Linux system with cpython<3.3, there could be abi tags 'm' and 'mu'
+    if 'linux' in platform and int(python_version[2:]) < 33:
+        dirname = os.path.join(dirname, 'linux')
+        # encode with ucs2
+        if sys.maxunicode == 65535:
+            dirname = os.path.join(dirname, python_version+'m')
+        # encode with ucs4
+        else:
+            dirname = os.path.join(dirname, python_version+'mu')
+
+    elif 'macosx' in platform:
+        platform = 'macosx'
+    dirname = os.path.join(dirname, platform)
+    sys.path = [os.path.realpath(str(dirname))] + sys.path
```

Comment:
Yes okay, obvious cause found :) I think the previous statement (overwritten line in 95d4a91eff29e507abb22019ba0802be136805f4) should have been a `sys.path.append()`, because concatenating a single-element list to another list is... well pretty unlikely... so at some point while developing the c extensions PR, we probably already had the c extensions path at the beginning of the list? Is there any specific concerns here to keep in mind why we might have done that @lyw07 ?

Reply:
oh I think at the time I wrote that line, I didn't know that the order mattered. Thank you for changing it though! I'll test this PR again soon!